MigrationTutorial/Models/V2/Employee.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using MongoDB.Bson;
+using Realms;
+
+namespace MigrationTutorial.Models.V2
+{
+    public class Employee : RealmObject
+    {
+        [PrimaryKey]
+        public ObjectId Id { get; private set; } = ObjectId.GenerateNewId();
+
+        [Required]
+        public string Fullname { get; set; }
+
+        [Required]
+        public int? Age { get; set; }
+
+        public Department Department { get; set; }
+
+        public Gender Gender
+        {
+            get => Enum.Parse<Gender>(_Gender);
+            set => _Gender = value.ToString();
+        }
+
+        private string _Gender { get; set; }
+
+        public Employee() { }
+    }
+
+    public enum Gender
```

Comment:
Would it be ok to add `NotSpecified` so that I can also address the following from @DominicFrei's [comment](https://github.com/LaPeste/migration-tutorial-src-temp/pull/1#discussion_r745556133)
That'd be used when people don't want to say or when the conversation can't say because of, perhaps, a misspelling.

Reply:
I think it's fine for this case.